### ISSUE TITLE
REQUEST: MyAnimeList support

### ISSUE THREAD

**Issue by pichu66** [nv95/OpenManga#41]
Hi!
Thanks for your hard work on this amazing app!
Can you add a sync with MyAnimeList ?
Thnks

**Comment by Koitharu** [comment]
I don't know about MAL's api. Is it exists?

**Comment by switchswap** [comment]
Yes, there's more info on it [here](https://myanimelist.net/modules.php?go=api).

**Comment by switchswap** [comment]
@nv95 Would you mind if I attempted to work on MAL integration?

**Comment by mangaMag** [comment]
MAL gives only info about the manga, i don't see any **use** to it but yeah it could be of some use

**Comment by switchswap** [comment]
I was thinking about a sort of MAL sync such that when you read a new chapter, or start a new series, your MAL will be updated accordingly. Either way, I'll attempt it in the coming week.

**Comment by Koitharu** [comment]
I think, it will be useful.
One problem: how to find manga on MAL by name (or something else)?

**Comment by switchswap** [comment]
MAL api is dead for now, so this is on hold. I'm switching to figuring out the AniList api.
Also, is this project still alive?

**Comment by Koitharu** [comment]
> MAL api is dead for now

So sad

> Also, is this project still alive?

Yes
